runtime_config_linux.go

```diff
@@ -200,20 +200,20 @@ type Network struct {
 type Resources struct {
 	// DisableOOMKiller disables the OOM killer for out of memory conditions
 	DisableOOMKiller bool `json:"disableOOMKiller"`
```

Comment:
This is more for the runtime implementation. nil means don't touch it. Even if the default is false, the runtime will have to go and overwrite the file if it isn't nil.